testutil/sample/sample.go

```diff
@@ -17,6 +17,14 @@ func AccAddressAndPubKey() (string, cryptotypes.PubKey) {
 	return cosmostypes.AccAddress(addr).String(), pk
 }
 
+// AccAddressAndKeyPair returns a sample account address its public key and private key
+func AccAddressAndKeyPair() (string, cryptotypes.PubKey, cryptotypes.PrivKey) {
+	sk := secp256k1.GenPrivKey()
```

Comment:
Can you please use this function inside of `AccAddressAndPubKey` so its a wrapper rather than duplicate logic?